libs/sdk-ui-filters/src/DateFilter/index.ts

```diff
@@ -74,3 +74,17 @@ export {
 
 export { defaultDateFilterOptions } from "./constants/config";
 export { GranularityIntlKey } from "./constants/i18n";
+
+// internal parts of DateFilter reused in AD DateFilter component
```

Comment:
I'm not sure if we should expose it, if it's marked as internal. I have some bad experience with this, that customers are then importing this internal stuff. IMO full imports in AD are a better option.

Reply:
Fixed. I removed the reexports and left just @internal tag on the components that will be reused by AD.